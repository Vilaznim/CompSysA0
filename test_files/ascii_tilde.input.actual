test_files/ascii_tilde.input: ASCII text
