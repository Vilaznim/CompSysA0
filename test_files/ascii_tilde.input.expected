test_files/ascii_tilde.input: very short file (no magic)
